data/brands/leisure/fitness_centre.json

```
@@ -72,6 +72,16 @@
         "sport": "kickboxing"
       }
     },
+    {
+      "displayName": "Activ Fitness",
```

Since it's probably a common mistake, we can add a match name.

```suggestion
      "displayName": "Activ Fitness",
      "matchNames": ["Active Fitness"],
```

That way iD will ask users if [POIs with the spelling mistake](https://www.openstreetmap.org/node/7503188456) need updating.

---

I didn't know that this is possible.
Then I would not have made https://www.openstreetmap.org/changeset/146830770 this afternoon, in which I corrected the wrong name of some Active Fitnesses to Activ Fitnesses.